tiup/tiup-playground.md

```diff
@@ -60,6 +60,12 @@ Flags:
       --tiflash.binpath string   Specify the TiFlash instance binary path (optional, for debugging)
       --tiflash.config string    Specify the TiFlash instance configuration file (optional, for debugging)
       --tiflash.timeout int      Specify TiFlash maximum wait time in seconds for starting. 0 means no limit
+      --tiproxy int              TiProxy instance number
+      --tiproxy.binpath string   TiProxy instance binary path
+      --tiproxy.config string    TiProxy instance configuration file
+      --tiproxy.host host        Playground TiProxy host. If not provided, TiProxy will still use host flag as its host
+      --tiproxy.port int         Playground TiProxy port. If not provided, TiProxy will use 6000 as its port
+      --tiproxy.timeout int      TiProxy max wait time in seconds for starting. 0 means no limit (default 60)
```

Comment:
```suggestion
      --tiproxy int              Specify the TiProxy instance number
      --tiproxy.binpath string   Specify the TiProxy instance binary path
      --tiproxy.config string    Specify the TiProxy instance configuration file
      --tiproxy.host host        Specify the Playground TiProxy host. If not provided, TiProxy will still use host flag as its host
      --tiproxy.port int         Specify the Playground TiProxy port. If not provided, TiProxy will use 6000 as its port
      --tiproxy.timeout int      Specify the TiProxy max wait time in seconds for starting. 0 means no limit (default 60)
```

Reply:
This should not be modified. It is the output of tiup. We should keep it as is.